### DISCUSSION TITLE
Unable to build protoc-c under Windows / MSYS2 / MinGW

### DISCUSSION THREAD

**Discussion by dlindelof** [protobuf-c/protobuf-c#624] [Q&A] [answered]
I'm interested in building `protobuf-c` for Windows with MSYS2 and MinGW64, so I can run `protoc-c` directly from the command line. I run the usual `./autogen.sh` and `./configure --prefix=${PWD}/install` successfully, but when I then try to run `protoc-c` I don't get the same behaviour as under Linux or macOS:

    $ install/bin/protoc-c

just hangs and shows nothing, while

    $ install/bin/protoc-c  --c_out=.

yields

    C:\msys64\home\dlindelof\...\protobuf-c\install\bin\protoc-c: Unknown option: --c_out=.

Any idea what might be the problem here? (I'm building from the `v1.3.1` tag)

**Comment by konosubakonoakua** [comment, marked as answer]
Hello, I install **mingw-w64-protobuf-c** following instructions [here](https://github.com/Alexpux/MINGW-packages).
And, also, I have the same problem using **protoc-c.exe**(**protoc.exe** functionally working), until I add environment variables **msys2/mingw64/bin**.
![snipaste_2019-01-11_15-15-12](https://user-images.githubusercontent.com/42881610/51018839-fbd6d300-15b3-11e9-9973-cc91db15730f.png)
Now, it works.
![snipaste_2019-01-11_15-14-53](https://user-images.githubusercontent.com/42881610/51018902-2aed4480-15b4-11e9-94f6-1425269ba853.png)
Besides, both files generated with **protoc.exe** or **protoc-c.exe** are the same.
I think we can just use **protoc.exe**, since it's smaller and working  **without** .dll file.
